src/plugins/console/server/lib/spec_definitions/js/mappings.ts

```diff
@@ -84,7 +85,20 @@ export const mappings = (specService: SpecDefinitionsService) => {
           // index_prefixes: { min_chars, max_chars },
 
           index_options: {
-            __one_of: ['docs', 'freqs', 'positions'],
+            __one_of: [
+              '',
+              // text-based types
+              { __one_of: ['positions', 'docs', 'freqs', 'offsets'] },
```

Comment:
`positions` has been default since 8.x - adjusted according to https://www.elastic.co/guide/en/elasticsearch/reference/8.15/index-options.html